packages/itwin/ec3-widget/src/components/templateClient.tsx

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Guid } from "@itwin/core-bentley";
+import { Template, Label, Material } from "./Template"
+
+// For now only one template for each report.
+
+export default class TemplateClient {
```

Comment:
File name does not match the class name. Rename the file (make the `t` upper case).

Reply:
File has been renamed